gapic/templates/tests/unit/gapic/%name_%version/%sub/test_macros.j2

```diff
@@ -2242,6 +2240,11 @@ def test_initialize_client_w_{{transport_name}}():
         {% endif %}{# method.server_streaming #}
         req.return_value = response_value
         response = {{ await_prefix }}client.{{ method_name }}(request)
+    {% if "next_page_token" in method_output.fields.values()|map(attribute='name', default="") and not method.paged_result_field %}
+    {# Cheeser assertion to force code coverage for bad paginated methods #}
```

Comment:
Please could you add more detail to help better understand the bad paginated methods?

Reply:
This is copied from another test within the same file, which I've used as reference for this macro. This specific assertion is rarely generated. I'm unsure what these bad paginated methods are and don't think we should spend time investigating on it given the approaching deadline. I've filed https://github.com/googleapis/gapic-generator-python/issues/2199 and linked the bug here to follow up on it.